packages/kube-object/src/kube-object.test.ts

```diff
@@ -0,0 +1,116 @@
+import { KubeObject } from "./kube-object";
+
+describe("kube object tests", () => {
+  it("should allow an object to be created when missing uid", () => {
```

Comment:
These technical tests should demonstrate why `uuid` and `resourceVersion` are needed in a `KubeObject`, and also what is different when they are missing.

Reply:
okay I can add tests to cover those cases too